src/uct/ib/base/ib_md.c

```diff
@@ -669,7 +698,7 @@ uct_ib_mem_reg_internal(uct_md_h uct_md, void *address, size_t length,
               params->dmabuf_offset);
 
     if (md->config.odp.prefetch) {
-        md->ops->mem_prefetch(md, memh, address, length);
+        uct_ib_mem_prefetch(md, memh, address, length);
```

Comment:
No rc checking here - intentional?

Reply:
Yes, we don't want to fail registration just because prefetch failed. Also, it was the previous behavior